R/netcoupler.R

```diff
@@ -0,0 +1,125 @@
+
+net.coupler <- function(Graph, Graph_data, Exp_data_SC, exposure) {
```

Comment:
There were duplicates of these functions. I kept any possible differences as comments, just in case. See individual commits for more detail.